saleor/order/actions.py

```diff
@@ -1120,6 +1121,19 @@ def automatically_fulfill_digital_lines(
         update_order_status(order)
 
 
+def _check_if_fulfillment_quantity_exceeds_order_quantity(order_line, quantity):
+    if order_line.quantity_unfulfilled - quantity < 0:
+        return True
+
+    allocated = order_line.allocations.aggregate(allocated=Sum("quantity_allocated"))[
```

Comment:
`_check_if_fulfillment_quantity_exceeds_order_quantity` is called for each order_line. Does it mean that we will make allocations for each line, right? So having many order_lines will cause that for each of them we will make an additional db query. Maybe good would be fetching the allocation first for all lines, then passing it to the function? 
Do I miss something? 🤔